jsapp/js/account/organization/organizationQuery.ts

```diff
@@ -95,25 +93,12 @@ export const useOrganizationQuery = (params?: OrganizationQueryParams) => {
 
   // Using a separated function to fetch the organization data to prevent
   // feature flag dependencies from being added to the hook
-  const fetchOrganization = async (): Promise<Organization> => {
+  const fetchOrganization = async (): Promise<Organization> =>
     // `organizationUrl` is a full url with protocol and domain name, so we're
```

Comment:
Just a style issue: I tend to think we should move all the doc text for fetchOrganization above it. It's essentially a single line of code broken for screen length purposes. Seeing that solo await without brackets around it feels weird.

Reply:
So, combining both comments... Revisiting the existing comment above `fetchOrganization`, seems if we're removing the feature flag, we can simplify it even more by putting `fetchGet<...>` into queryFn below

Reply:
Thank you...nice catch. I visually ignored those comments and should've just removed everything at once. Just did it now. Also some changes to the query were required due to dependencies, since now we're using the `organizationUrl` inside of the query.